api/controllers/likeController.js

```diff
@@ -0,0 +1,62 @@
+const { likeService } = require('../services');
+const { CustomError } = require('../utils/error');
+
+const checkLike = async (req, res) => {
+    try {
+        const { productId } = req.params;
+        const userId = req.user;
+        const result = await likeService.checkLike(userId, productId);
+
+        if (result) {
+            return res.status(201).json({ isLiked: true });
+        }
+
+        if (!result) {
+            return res.status(201).json({ isLiked: false });
+        }
```

Comment:
```suggestion
    try {
        const { productId } = req.params;
        const userId = req.user;
        const result = await likeService.checkLike(userId, productId);

        if (!result) res.status(201).json({ isLiked: false });
        
        return res.status(201).json({ isLiked: true });
```
어떠한 차이가 있는지 비교해보시면 좋을 것 같아요 !
또한 해당 API는 어떠한 data를 변형시키는 것이 아닌, 단순히 check만 하는 것 같아 보다 더 적절한 status code가 쓰여야 할 것 같습니다.